accounts/abi/bind/util_test.go

```diff
@@ -33,6 +33,9 @@ import (
 
 var testKey, _ = crypto.HexToECDSA("b71c71a67e1177ad4e901695e1b4b9ee17ae16c6668d313eac2f96dbcda3f291")
 
+// TODO(Joshua): Replace with call to GPM or faked GPM
```

Comment:
Should this be done before merging? I'm assuming not, but asking just in case

Reply:
Hmm, in this case the it's basically setting a constant gas price and working. I don't think it needs to be reactive to the GPM.